tests/legacy-cli/e2e/tests/misc/completion-prompt.ts

```diff
@@ -19,6 +19,11 @@ const DEFAULT_ENV = Object.freeze({
 });
 
 export default async function () {
+  if (process.platform === 'win32') {
+    await windowsTests();
+    return; // Windows should never prompt for autocompletion.
```

Comment:
Would this fail if the test was run on Windows with git-bash?
CircleCI is setup to use powershell so CI would be fine. maybe expand the comment to note that.

Reply:
Yes, I think this would fail for Git Bash, I wasn't too concerned since CI isn't doing that.

It kinda sucks since any contributors using Git Bash will run the wrong set of tests and possibly get false negatives. I'm not too worried about it right now though, and I think we can figure that out when someone actually tries to do it, resolves all the other issues they'd run into with Git Bash in our setup (no idea how it plays with Bazel for instance), and can more easily find the right detection mechanism.